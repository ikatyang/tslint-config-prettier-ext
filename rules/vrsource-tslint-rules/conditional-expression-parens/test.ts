let x = (y > 30) ? "left" : "right";
