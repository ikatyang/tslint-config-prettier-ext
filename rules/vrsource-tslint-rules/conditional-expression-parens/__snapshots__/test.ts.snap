// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
no error

======
ERROR: (conditional-expression-parens) /rules/vrsource-tslint-rules/conditional-expression-parens/test.ts[1, 9]: Expression clause must be parenthesized

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
let x = (y > 30) ? \\"left\\" : \\"right\\";

======
let x = y > 30 ? \\"left\\" : \\"right\\";

>>>>>> after
"
`;
